devices/Mcp23xxx/PinValuePairEnum.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections;
+
+namespace Iot.Device.Mcp23xxx
+{
+    public class PinValuePairEnum : IEnumerator
```

Comment:
if this class is public, please adjust it to add the intellisense comments ///
It has to be on everything public or internal

Reply:
file not necessary, removed from Mcp23xxx project